cover_my_meds.gemspec

```diff
@@ -12,26 +10,23 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{CoverMyMeds Public API}
   spec.description   = %q{The public version of CoverMyMeds API}
   spec.homepage      = "https://github.com/covermymeds/cover_my_meds"
-
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler"
-  spec.add_development_dependency "rspec", ">= 3.2"
-  spec.add_development_dependency "rspec-junklet", ">= 2.0"
-  spec.add_development_dependency "webmock", "~> 2.0"
+  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rspec-junklet"
+  spec.add_development_dependency "webmock"
```

Comment:
I think we can forego versioning on dev dependencies unless we're really locked into a version.